sdk/resourcemanager/azure-resourcemanager-eventhubs/src/main/java/com/azure/resourcemanager/eventhubs/models/EventHubNamespace.java

```diff
@@ -220,6 +230,32 @@ interface WithThroughputConfiguration {
             WithCreate withThroughputUnitsUpperLimit(int units);
         }
 
+        /**
+         * The stage of the event hub namespace definition
+         * allowing to specify minimumTlsVersion.
+         */
+        interface WithMinimumTlsVersion {
+            /**
+             * Specifies that minimum TLS version should be created in the namespace.
+             *
+             * @param minimumTlsVersion minimum TLS version
+             * @return next stage of the event hub namespace definition
+             */
+            WithCreate withMinimumTlsVersion(TlsVersion minimumTlsVersion);
+        }
+
+        /**
+         * The stage of the event hub namespace definition allowing to enable Zone Redundant.
+         */
+        interface WithZoneRedundant {
+            /**
+             * Enables Zone Redundant.
+             *
+             * @return next stage of the event hub namespace definition
+             */
+            WithCreate enableZoneRedundant();
```

Comment:
Is this updatable? 

Maybe not from true to false, but can it be updated from false to true?

Reply:
Show me the source of this "immutable variable" assertion?

Reply:
This is the conclusion drawn after executing the test. the test result following as:
[test.log](https://github.com/user-attachments/files/16246967/test.log)

Reply:
OK, if test result, just say test result is such.

Don't say something like it is documented by service, if it is not. Behavior of your test can change anytime whenever backend changes.